internal/csi-common/server.go

```diff
@@ -24,8 +24,8 @@ import (
 	"sync"
 
 	"github.com/container-storage-interface/spec/lib/go/csi"
-	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
-	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
+	grpcmiddleware "github.com/grpc-ecosystem/go-grpc-middleware"
+	grpcprometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
```

Comment:
This is from the package name https://github.com/grpc-ecosystem/go-grpc-prometheus/blob/9abf3eb82b4a313b1a28f370a7ef8ffe0667c681/server_metrics.go#L1 any specific reason to change this one?

Reply:
@Madhu-1 the package name can be in that format, havent seen the import name in snake case, have you seen such occurrence?

Reply:
the sample code was referred from https://pkg.go.dev/github.com/grpc-ecosystem/go-grpc-middleware. if we want to remove it we need to document it in development guide to avoid such occurrence or we can remove the alias let us the package name directly

Reply:
Let me rework the patch. There are 3 more instances to be changed. @Madhu-1

Reply:
> snake case and camel cases are not recommended though

Where is this documented? If there are generic Golang guidelines that we should follow, it would be best to also enable checking for them in the golang-ci configuration.

Reply:
@nixpanic I know this is implemented in projects like `kubernetes` where camelcase, snake case imports are not allowed.  Also see https://rakyll.org/style-packages/. Is it enough proof for removing such instances from this repo?

Reply:
@nixpanic @Madhu-1 updated all the instances and also updated the commit message. PTAL.

Reply:
I would like to know how we can prevent this from getting re-introduced in the future. Is there a golang-ci option that we can enable after this commit?

Reply:
@nixpanic sure,, I can check whether we can emphasize in the CI somehow and file a PR accordingly.. Can we get this in ?

Reply:
I think I prefer to see a CI job in combination with this PR. Can you add that as a 2nd commit?

Reply:
@nixpanic I still dont know how to catch such instances in CI with a rule . Thats why I said, I have to check on it.

Reply:
It is not urgent to have this addressed, so it is fine to keep this PR open until there is a CI job to validate this change.